static/bidder-info/connectad.yaml

```diff
@@ -1,16 +1,29 @@
+# Please uncomment the appropriate endpoint URL for your datacenter
+# Europe
 endpoint: "http://bidder.connectad.io/API?src=pbs"
+# North/South America
+# endpoint: "http://bidder-us.connectad.io/API?src=pbs"
+# APAC
+# endpoint: "http://bidder-apac.connectad.io/API?src=pbs"
+geoscope:
+  - global
 maintainer:
   email: "support@connectad.io"
+endpointCompression: gzip
 gvlVendorID: 138
 capabilities:
   app:
     mediaTypes:
-    - banner
+      - banner
   site:
     mediaTypes:
-    - banner
+      - banner
 userSync:
-  iframe:
-    url: "https://cdn.connectad.io/connectmyusers.php?gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&cb={{.RedirectURL}}"
+  redirect:
+    url: "https://sync.connectad.io/ImageSyncer?gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&cb={{.RedirectURL}}"
     userMacro: ""
     # connectad appends the user id to end of the redirect url and does not utilize a macro
+  iframe:
+    url: "https://sync.connectad.io/iFrameSyncer?gdpr={{.GDPR}}&consent={{.GDPRConsent}}&us_privacy={{.USPrivacy}}&gpp={{.GPP}}&gpp_sid={{.GPPSID}}&cb={{.RedirectURL}}"
```

Comment:
![Screenshot 2024-06-17 at 1 13 43 PM](https://github.com/prebid/prebid-server/assets/8147531/d7ace66f-3b0e-4fea-8033-effacd70f0bd)

Getting 403 for userSync. Please check.

```
https://sync.connectad.io/iFrameSyncer?gdpr=&consent=&us_privacy=&gpp=&gpp_sid=&cb=https%3A%2F%2Fib.adnxs.com%2Fprebid%2Fsetuid%3Fbidder%3Dconnectad%26gdpr%3D%26gdpr_consent%3D%26gpp%3D%26gpp_sid%3D%26f%3Db%26uid%3D
```

Reply:
I am also getting 403

Reply:
are you trying to access that url via VPN? Where are you located? both sync urls are up and running

Reply:
We are trying from India.

Reply:
As long as you are not calling it from a suspicious, banned, or datacenter ip, there will be no issue @gargcreation1992

Reply:
Checked again. Worked fine this time.